opta/core/kubernetes.py

```diff
@@ -44,6 +45,22 @@
 GCP_CLI_INSTALL_URL = "https://cloud.google.com/sdk/docs/install"
 
 
+register_path_executable("kubectl", install_url=KUBECTL_INSTALL_URL)
+register_path_executable("aws", install_url=AWS_CLI_INSTALL_URL)
+register_path_executable("gcloud", install_url=GCP_CLI_INSTALL_URL)
+register_path_executable("az")
```

Comment:
install url?

Reply:
There wasn't one present originally, but I'll look it up and add it.